cmd/antrea-agent/agent.go

```diff
@@ -220,10 +220,10 @@ func run(o *Options) error {
 	v4GroupCounter := proxytypes.NewGroupCounter(false, groupIDUpdates)
 	v6GroupCounter := proxytypes.NewGroupCounter(true, groupIDUpdates)
 
+	v4Enabled := agentInitializer.IPv4Enabled
```

Comment:
But they are not used outside of the if features.DefaultFeatureGate.Enabled(features.AntreaProxy) {} block. Maybe you want to change line 448 and 449 to use these two variables?

Reply:
These two variables are actually used inside "if features.DefaultFeatureGate.Enabled(features.AntreaProxy) {} block". You are correct, I would change line 448 to use these two.